src/index.js

```diff
@@ -22,6 +23,14 @@ export class AddonStore {
     this._preview = preview;
   }
 
+  getDatabase() {
+    return this._database;
+  }
+
+  setDatabase(database) {
```

Comment:
Addons are installed by users so if they're installing an add-on which changes the DB it's probably because they want it to do that. So it's not up to us.

Addons like comments don't see where the data gets stored. They only see a common interface (collections, set, get).

Consider these cases:
By default the database is provided by a flat file DB which will store data in the repo itself. Services like storybooks.io can add a custom storage add-on for its users. If someone wants to store their in their own servers they can write and add a storage addon.